pkg/agent/multicast/kernel_source.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 Antrea Authors
```

Comment:
What does the file name kernel_source mean? Probably you should add some comments at the beginning of the file to explain.